tools/hclust2/hclust2.xml

```diff
@@ -0,0 +1,562 @@
+<tool id="hclust2" name="hclust2" version="@VERSION@.0">
+    <description>plots heatmaps</description>
+    <macros>
+        <token name="@VERSION@">0.99</token>
+    </macros>
+    <requirements>
+        <requirement type="package" version="@VERSION@">hclust2</requirement>
+    </requirements>
+    <command detect_errors="exit_code">
+    <![CDATA[
+    hclust2.py
+	    -i '$i'
+	    #if $o == "png"
+			-o 'output.png'
```

Comment:
@yuanbit can you check the indentation so that you are using 4 spaces instead of tabs.

Reply:
@bgruening Thank you for the comments. The indentations should be fixed now!